lib/table.py

```diff
@@ -110,7 +112,11 @@ def combine_on_delimiter(self, line_as_list):
         """Combine a list of values into a line of csv data"""
         dialect = csv.excel
         dialect.escapechar = "\\"
-        writer_file =  io.BytesIO()
+        if sys.version_info >= (3, 0):
+            writer_file =  io.StringIO()
+        else:
+            writer_file =  io.BytesIO()
```

Comment:
@ethanwhite This code gives an error in Python3:
```
line_as_list = [line.encode() for line in line_as_list]
writer_file =  io.BytesIO()
writer = csv.writer(writer_file, dialect=dialect, delimiter=self.delimiter)
for line in line_as_list:
    assert isinstance(line,bytes)
    writer.writerow(line)```
Error: 
```
>           writer.writerow(line)
E           TypeError: a bytes-like object is required, not 'str'
```
So I had to use `io.StringIO` for Python3. I have no idea why csv can't understand `bytes` type when it sees it (as the assert statement passes without error).